DNN Platform/Components/ClientDependency/Source/DependencySorter.cs

```diff
@@ -10,6 +10,30 @@ namespace ClientDependency.Core
     /// </summary>
     internal static class DependencySorter
     {
+
+        public static IList<IClientDependencyFile> FilterDependencies(IList<IClientDependencyFile> dependencies)
+        {
+            if (dependencies.Any(f => f.Name != ""))
+            {
+                var newList = dependencies.Where(f => f.Name == "").ToList();
+                var frameworks = dependencies.Where(f => f.Name != "").GroupBy(f => f.Name.ToLower());
+                foreach (var framework in frameworks)
+                {
+                    var topPriority = framework.FirstOrDefault(d => d.ForceVersion);
```

Comment:
This property is not available through the Client Resource Manager methods, but only in the control. This effectively makes it only available to skin developers. So you're down to one person.

If someone decides to "hack" or abuse the feature, the result is unpredictable. But at the end of the day it's down to instruction: this is only meant for Skin designers, not the modules.